File: quest/arrays/02_shuffle.py
class Solution(object):
    def shuffle(self, nums, n):
        """
        :type nums: List[int]
        :type n: int
        :rtype: List[int]
        """
        
        ans = [0] * (2 * n)
        for i in range(n):
            ans[i * 2] = nums[i]
            ans[i * 2 + 1] = nums[i + n]

        return ans

File: quest/arrays/02_shuffle.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
    var ans = new Array(n * 2);
    for (i = 0; i < n; i++) {
        ans[i * 2] = nums[i];
        ans[(i * 2) + 1] = nums[i + n];
    }

    return ans;
};
